packages/ffe-account-selector-react/src/components/account-selector/AccountSelector.js

```diff
@@ -96,6 +96,7 @@ class AccountSelector extends Component {
                     {...this.props}
                     onSelect={this.onAccountSelect}
                     onChange={this.onInputChange}
+                    locale={locale}
```

Comment:
Just for reference in case others are wondering: `AccountSelectorMulti` already passed on `locale`